### PR TITLE
feature/#1-dice-thrower

### LINKED ISSUE #1: Feature/#1 - Logic - Dice generator
Create a Die class with:
- [x] a constructor that takes an array with the decomposition of sides of the die 
- [x] a method that in result will give us a random result of a die roll
- [x] create an interface with getRandomResult method

Remember, that decomposition is as follows:
Dice 1: 6x Rabbit, 2x Sheep, 2x Pig, 1x Horse, 1x Fox
Dice 2: 6x Rabbit, 3x Sheep, 1x Pig, 1x Cow, 1x Wolf

### PR DESCRIPTION
**Pull request by synowa**
closes #1

### CODE REVIEW

File: src/app/Dice.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { GetRandomValue } from './Interfaces/DiceInterface';
+import { AnimalNames } from './Enums/AnimalNamesEnum';
+
+class Dice implements GetRandomValue {
+  constructor(private diceSides: AnimalNames[]) {}
+  getRandomValue(): AnimalNames {
+    return this.diceSides[Math.floor(Math.random() * 12)];
```
Comment (Enessetere) [inline src/app/Dice.ts]: Consider using lodash and its 'sample' method.

**Review by Enessetere — APPROVED** [review]
Looks good to me, well done. Consider change I have mentioned.

**Review by SebastianBabinski1 — APPROVED** [review]
Looks good, consider changes proposed by Sebastian M :)

File: src/app/Dice.ts (hunk 1 above)
Comment (synowa) [reply src/app/Dice.ts]: Ok, thanks a lot for your suggestion!

Comment (DominikNowak) [reply src/app/Dice.ts]: You can also consider shuffling array in constructor for better randomization. There is function for that in lodash named 'shuffle'. I don't see also sense of classes FirstDice and SecondDice. Arrays with default sides should be somewhere in constants. But time is running out so let's leave these for future refactor.

**Pull request merged**